### PR TITLE
fix(ui): Add module to issue owners examples

### PR DESCRIPTION
**Pull request by taylangocmen**
This adds module as an example to the issue owners. Module is supported in the Docs: [Issue Ownership](https://docs.sentry.io/product/issues/issue-owners/#ownership-rules) and in the BE: [sentry/grammar.py](https://github.com/getsentry/sentry/blob/34e9dc0e5a65fa606d679aabf5e10af03b2cb0d2/src/sentry/ownership/grammar.py#L127) but wasn't visible in the FE examples and dropdown. Added for parity

Jira: [WOR-1903](https://getsentry.atlassian.net/browse/WOR-1903)

After:
Screen Shot 2022-06-09 at 4 07 33 PM

Before:
Screen Shot 2022-06-09 at 4 07 24 PM

### CODE REVIEW

**Review by barkbarkimashark — APPROVED** [review]

File: static/app/views/settings/project/projectOwnership/editRulesModal.tsx [hunk 1]
```diff
@@ -27,7 +27,9 @@ class EditOwnershipRulesModal extends Component<Props, State> {
         <Block>
           {t('Examples')}
           <CodeBlock>
-            path:src/example/pipeline/* person@sentry.io #infrastructure
```
Comment (barkbarkimashark) [inline static/app/views/settings/project/projectOwnership/editRulesModal.tsx:30]: why the change from `#infrastructure` to `#infra`

Comment (taylangocmen) [reply static/app/views/settings/project/projectOwnership/editRulesModal.tsx:30]: It's `#infra` in two other places
